packages/paste-website/src/pages/utilities/grid/index.mdx

```diff
@@ -0,0 +1,533 @@
+---
+title: Grid - Utilities
+description: The grid component controls the visual layout and spacing between elements on a page or in containers.
+---
+
+import {graphql} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import {Grid, Column} from '@twilio-paste/grid';
+import {Card} from '@twilio-paste/card';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Text} from '@twilio-paste/text';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {SiteLink} from '../../../components/SiteLink';
+import Changelog from '@twilio-paste/grid/CHANGELOG.md';
+import {TwilioIcon} from '../../../components/icons/TwilioIcon';
+import {TWILIO_RED} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteUtility(filter: {name: {eq: "@twilio-paste/grid"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/grid/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Grid"
+  categoryRoute={SidebarCategoryRoutes.UTILITIES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/utilities/grid"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/utilities-grid--grid-12-column-and-gutter-support"
+  abstractUrl="https://share.goabstract.com/3e7528f5-7981-421c-9308-e06d2f341c70?"
+  data={props.data.allPasteUtility.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+### About Grid
+
+Grid creates organized and consistent product layouts, by providing a standard for positioning elements. Grid speeds up the designer-to-developer workflow by improving the consistency of design implementations.
+
+Use a Grid to layout and align content by applying standard spacing between elements on a page or within a component.
+
+The standard desktop grid has 24-pixel gutters and can support up to 12 columns. The outside gutters are removed from the overall width of the grid.
+
+Grid is built using the [Flex component](/utilities/flex) and is responsive. Grid column widths grow and shrink based on the parent element width.
+
+## Examples
+
+#### Equal column widths
```

Comment:
You’re skipping a heading level for these examples. Make sure we adhere to a good document hierarchy